test/specs/scenarios/proofOfAddress.js

```diff
@@ -222,11 +222,9 @@ export const proofOfAddressScenarios = async (lang = 'en_US') => {
 
       it('should succesfully complete cross device e2e flow with PoA document and selfie upload', async () => {
         const copyCrossDeviceLinkAndOpenInNewTab = async () => {
-          /* eslint-disable indent */
```

Comment:
All these `eslint-disable indent` are weird!

Reply:
These toggles before as they were getting flagged during the lint check in builds, but I think the additional rules @ethan605 added recently covered them.